jvb/rootfs/opt/jitsi/shutdown.sh

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/with-contenv bash
+# notify the sidecar of imminent shutdown
+[ -z "$AUTOSCALER_SIDECAR_PORT" ] && export AUTOSCALER_SIDECAR_PORT="6000"
```

Comment:
No need to export it since there is no subshell. How about `PORT=${AUTOSCALER_SIDECAR_PORT:-6000}` and then use $PORT.

Reply:
agreed